srcpkgs/nano-syntax-highlighting/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'nano-syntax-highlighting'
+pkgname=nano-syntax-highlighting
+version=2025.07.01
+revision=1
+depends="nano"
+short_desc="Nano editor syntax highlighting enhancements"
+maintainer="masn <masn@togeter.net>"
+license="GPL-3.0-or-later"
```

Comment:
The license is something custom and not GPL3: https://github.com/galenguyer/nano-syntax-highlighting/blob/master/license

Reply:
Oh, my bad, it's actually GPL3-or-later, I was just confused because they didn't include the normal GPL file. So you can just leave it as it is.